store/v2/root/factory.go

```diff
@@ -50,7 +54,7 @@ func DefaultStoreOptions() Options {
 			Interval:   100,
 		},
 		IavlConfig: &iavl.Config{
-			CacheSize:              100_000,
+			CacheSize:              1_000_000,
```

Comment:
we may want to revert this, 1m could be a lot for memory. node operators can override to set higher

Reply:
https://github.com/cosmos/cosmos-sdk/blob/eb3bf8b0469d0185a7c3c323c8888e691a6b62ff/store/iavl/store.go#L22-L24

maybe set to 500k, the default in store/v1?

Reply:
works